lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -753,11 +778,14 @@ global proc addMayaReferenceToUsdUiCb(string $parent, string $selectedFile)
     global int $gLastOptionSelectionIndex;
     global string $gLastOptionSelection;
     global string $gCurrentOptionSelection;
+    global string $gUsdMayaReferencePrimName;
 
     string $fileType;
     string $optionsScript;
 
     // Save the various Usd Maya Reference Options.
+    $gUsdMayaReferencePrimName = `textFieldGrp -query -text usdMayaReferencePrimName`;
```

Comment:
Using a global var to store the result from the field (for use by the callback) as no persistence was requested.